src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -100,7 +102,8 @@ namespace winrt::TerminalApp::implementation
             auto otherAsUs = other.try_as<CopyTextArgs>();
             if (otherAsUs)
             {
-                return otherAsUs->_SingleLine == _SingleLine;
+                return otherAsUs->_SingleLine == _SingleLine &&
+                       otherAsUs->_CopyFormatting == _CopyFormatting;

```

Comment:
does `==` do what we think it does for IReference?

Reply:
I assume yes? We do the same thing in a few places in this file:
- `SetTabColorArgs`'s `TabColor` (`IReference<uint32_t>`)
- `NewTerminalArgs`'s `ProfileIndex` (`IReference<int32_t>`)